wiki/osu!tourney/en.md

```diff
@@ -296,14 +292,13 @@ We may award badges to the first place winners of second (and further) iteration
 
 ## Troubleshooting
 
-### How do fresh install osu! without uninstalling my current?
+### How do I fresh install osu! without uninstalling my current?
 
-Copy-paste the `osu!.exe` into an empty folder and run it.
+Copy the `osu!.exe` into an empty folder and run it.
 
 ### My osu! windows are not aligning properly!
 
-**Please disable any secondary monitors!**
-
+Disable any **secondary monitors!**
 osu!tourney only runs on the primary monitor.
 A small screen resolution could also cause this problem as well.
```

Comment:
Let's reword this one to "osu!tourney only runs on the primary monitor. Try disabling any secondary monitors you have active. Make sure that the primary monitor has a resolution larger than the resolution of the client (1280x720 by default).